doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,223 @@
+specifying the ``sigma`` and ``patch radius`` values and ``output directory``.
+
+Create a directory where to save the denoised image (e.g.:
+``denoise_nlmeans_output``)::
+
+Then, we will run the command as::
+
+    dipy_denoise_nlmeans data/cfin_multib/__DTI_AX_ep2d_2_5_iso_33d_20141015095334_4.nii --sigma 2 --patch_radius 2 --out_dir "denoise_nlmeans_output"
+
+The command will denoise the input diffusion volume and write the result to the specified
+output directory.
+
+.. |image5| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_original.png  
+   :scale: 20%
+   :align: middle
+.. |image6| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_denoise_NLMEANS.png
+   :scale: 20%
+   :align: middle
+
++--------------------+--------------------+
+|  Before Denoising  |  After Denoising   |
++====================+====================+
+|      |image5|      |      |image6|      |
++--------------------+--------------------+
+
+-----------------------------
+Overview of Denoising Methods
+-----------------------------
+
+.. |image7| image:: https://github.com/dipy/dipy_data/blob/master/sherbrooke_3shell_original.png  
+   :scale: 100%
+   :align: middle
+.. |image8| image:: https://github.com/dipy/dipy_data/blob/master/sherbrooke_denoise_LPCA.png
+   :scale: 100%
+   :align: middle
+.. |image9| image:: https://github.com/dipy/dipy_data/blob/master/sherbrooke_3shell_denoise_MPPCA.png  
+   :scale: 100%
+   :align: middle
+.. |image10| image:: https://github.com/dipy/dipy_data/blob/master/sherbrooke_denoise_NLMEANS.png
+   :scale: 100%
+   :align: middle
+.. |image11| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_original.png  
+   :scale: 100%
+   :align: middle
+.. |image12| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_denoise_LPCA.png
+   :scale: 100%
+   :align: middle
+.. |image13| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_denoise_MPPCA.png  
+   :scale: 100%
+   :align: middle
+.. |image14| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_denoise_NLMEANS.png
+   :scale: 100%
+   :align: middle
+.. |image15| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_original.png  
+   :scale: 100%
+   :align: middle
+.. |image16| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_LPCA.png
+   :scale: 100%
+   :align: middle
+.. |image17| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_denoise_MPPCA.png
+   :scale: 100%
+   :align: middle
+.. |image18| image:: https://github.com/dipy/dipy_data/blob/master/cfin_multib_denoise_NLMEANS.png
+   :scale: 100%
+   :align: middle
+.. |image19| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_t1_original.png
+   :scale: 100%
+   :align: middle
+.. |image20| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_t1_NLMEANS.png
+   :scale: 100%
+   :align: middle
```

Comment:
you need to add `?raw=true` at the end of the URL.

Reply:
Addressed in 12ceeef